docs/mor/modules/ROOT/pages/thermalbuilding/index.adoc

```diff
@@ -126,10 +128,50 @@ The table displays the various fixed and variables parameters of this test-case.
 .Table of constant parameters
 [width="100%"]
 |=======================================================================
-| Name         | Description                    | Range                 | Units
-| stem:[\gamma] | Boundary conditions using Nitsche method | stem:[[10]]      | 
-| stem:[\k_0] | Air conductivity     | stem:[1]      | 
-| stem:[\k_1] | Conductivity - internal walls | stem:[0.25]      | 
-| stem:[\k_2] | Conductivity - internal doors  | stem:[0.13]      |
-| stem:[\sigma]  | Heat transfer coefficient - front door | stem:[\frac{1.0}{0.06+\frac{0.06}{0.150}+\frac{0.1}{0.029}+0.14}] | 
-|=======================================================================
\ No newline at end of file
+| Name         | Description                    | Value                 
+| stem:[\gamma] | Boundary conditions using Nitsche method | stem:[10]
+| stem:[k_0] | Air conductivity     | stem:[1]
+| stem:[k_1] | Conductivity - internal walls | stem:[0.25]
+| stem:[k_2] | Conductivity - internal doors  | stem:[0.13]
+| stem:[\sigma]  | Heat transfer coefficient - front door | stem:[\frac{1.0}{0.06+\frac{0.06}{0.150}+\frac{0.1}{0.029}+0.14}]
+|=======================================================================
+
+
+== Running the case using a Jupyter notebook
+
+It is possible to download this page as a Jupyter notebook and run it in an environment that contains a local installation of {feelpp} and its Python wrappers. The offline creation of the reduced basis has already been performed, and an archive is downloaded from Girder. It contains the basis, the model and the configuration files that are necessary for the online simulation.
+
+.Download the archive from Girder
+[%dynamic,python]
+----
+import requests
+r=requests.get("https://girder.math.unistra.fr/api/v1/item/64d60522b0e9570499e1eaa1/download")
+with open('thermalbuilding.fpp','wb') as f:
+    f.write(r.content)
+----
+
+.Create the online model, choose randomly 4 parameter vectors and simulate them using a reduced basis of 10 elements
+[%dynamic,python]
+----
+import feelpp
+from feelpp.mor import *
+ms=feelpp.mor.MORModels("thermalbuilding.fpp")
+muspace = ms.parameterSpace()
+sampling = muspace.sampling()
+sampling.sample(4, "random")
+r=ms.run(sampling,{"N":10})
+----
+
+.Print the outputs and the associated parameter vector
```

Comment:
@lberti you you could create a panda data frame and use it to have a nice table?

Reply:
I have created a Pandas data frame. The output of this test case is still weird though